plugin/trino-elasticsearch/src/main/java/io/trino/plugin/elasticsearch/client/ElasticsearchClient.java

```diff
@@ -378,13 +378,13 @@ private static void validateCertificates(KeyStore keyStore)
     private Set<ElasticsearchNode> fetchNodes()
     {
         NodesResponse nodesResponse = doRequest("/_nodes/http", NODES_RESPONSE_CODEC::fromJson);
-
+        Set<String> nodeRoles = Set.of("data", "data_hot", "data_content");
```

Comment:
Please define it as a `private static final ` field in the class and use `ImmutableSet` instead of `Set` (current code convention)